services/movie.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import QuerySet
+
+from db.models import Actor, Genre, Movie
+
+
+def get_movies(genres_ids: list[Genre] = None,
+               actors_ids: list[Actor] = None
+               ) -> QuerySet:
```

Comment:
The return type of the function `get_movies` should be `QuerySet[Movie]` instead of just `QuerySet` to specify the type of objects contained in the QuerySet.